eq-author/src/App/page/Routing/RoutingPage/RoutingEditor/RuleEditor/BinaryExpressionEditor/MultipleChoiceAnswerOptionsSelector.js

```diff
@@ -134,31 +140,36 @@ class MultipleChoiceAnswerOptionsSelector extends React.Component {
           defaultValue={expression.condition}
           data-test="condition-dropdown"
         >
-          <option value="AllOf">All of</option>
-          <option value="AnyOf">Any of</option>
+          <option value="AllOf">{answerConditions.ALLOF}</option>
+          <option value="AnyOf">{answerConditions.ANYOF}</option>
+          <option value="Unanswered">{answerConditions.UNANSWERED}</option>
```

Comment:
```suggestion
          <option value={answerConditions.UNANSWERED}>{answerConditions.UNANSWERED}</option>
```

Reply:
Seems weird to do this when you cant do it for the other values?

Reply:
Good point, it should probably be the other way round e.g.
```
<option value={answerConditions.ALLOF>"All of"</option>
<option value={answerConditions.ANYOF}>"Any of"</option>
<option value={answerConditions.UNANSWERED}>"Unanswered"</option>
```